### PR TITLE
added header component

### PR DESCRIPTION
**Pull request by anuradha9712**

### CODE REVIEW

File: src/App.js [hunk 1]
```diff
@@ -2,16 +2,23 @@ import React from "react";
...
 import Footer from "./components/Footer.jsx";
+import Header from './components/Header.jsx';
+import 'bootstrap/dist/css/bootstrap.min.css';
+import logo from './logo.svg';
+
 
 function App() {
   return (
     <div className="App">
+      <Header width="30" height="30" logoImg={logo} heading="funkids"></Header>
```
Comment (vsasvipul0605) [inline src/App.js]: I think propname logoIcon will look more relatable.

Comment (vsasvipul0605) [inline src/App.js]: And navbar background light or dark or any other color if required. should also be passed as prop

File: src/components/Header.jsx [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+            <Navbar bg="dark" variant="dark">
+                <Navbar.Brand href="/Home" className="navBrand">
+                    <img
+                        alt=""
+                        src={logoImg}
+                        width={width ? width: 30}
+                        height={height ? height : 30}
+                        className="d-inline-block align-top"
+                    />{' '}
+                    {heading}
```
Comment (vsasvipul0605) [inline src/components/Header.jsx]: Also, add default Heading prop as LOGO or BRAND-NAME if anyone does not pass any value to heading prop in App.js

File: src/components/Header.jsx [hunk 3]
```diff
@@ -0,0 +1,36 @@
...
+                </Navbar.Brand>
+
+                <Navbar.Toggle aria-controls="basic-navbar-nav" />
+                <Navbar.Collapse id="basic-navbar-nav">
+                    <Nav className="ml-auto">
+                        <Nav.Link href="/Home">Home</Nav.Link>
+                        <Nav.Link href="/About">About Us</Nav.Link>
+                        <Nav.Link href="/Contact">Contact Us</Nav.Link>
```
Comment (vsasvipul0605) [inline src/components/Header.jsx]: These Nav-Links must also be dynamic. As this is not mandatory that Navbar must contain Home About Us and Contact Us.

**Review by vsasvipul0605 — CHANGES_REQUESTED** [review]

File: src/components/Header.jsx (hunk 3 above)
Comment (anuradha9712) [reply src/components/Header.jsx]: Thanks, @vsasvipul0605 for your suggestions. I'll make the required changes

**Comment by AkbarHabeeb** [comment]
@anuradha9712 Your PR was affecting the existing button options(Reverted the merge). Please pull the latest changes and add a screenshot showing the changes.

**Pull request merged**